### ISSUE TITLE
Makevars (and Makevars.win) should be portable

### ISSUE THREAD

**Issue by pcarbo** [stephenslab/mashr#52]
This warning should be fixed. (The simplest solution is to just list all the filenames by hand.)

```
Found the following file(s) containing GNU extensions:
  src/Makevars
Portable Makefiles do not use GNU extensions such as +=, :=, $(shell),
$(wildcard), ifeq ... endif. See section ‘Writing portable packages’ in
the ‘Writing R Extensions’ manual.
```

**Comment by gaow** [comment]
Hmm this is interesting. Where did you get that warning? 

We can certainly type in all files instead of using wildcard ... but the difficult lines are:

```
GSL_CFLAGS    = $(shell ${R_HOME}/bin/Rscript -e "RcppGSL:::CFlags()")
GSL_LIBS      = $(shell ${R_HOME}/bin/Rscript -e "RcppGSL:::LdFlags()")
```

do you have a happier solution? I've briefly glanced at using `autoconf` but I am not sure how exactly it works with Rcpp ...

**Comment by pcarbo** [comment]
Fixed in commit 83158e7.

**Comment by pcarbo** [comment]
> I've mixed feelings about re-organizing the code in particular putting them under the same src folder as the CPP interface.

I agree, it is better to put in a different folder, but in any case we will make it clear that mashr includes code from Jo Bovy (with a license, etc).

> I've got one question: now that you have Makevars.in as a template, Should we just remove Makevars and add it to gitignore?

Yes, I forgot to do that. Thanks.
